__tests__/topics.test.js

```diff
@@ -0,0 +1,34 @@
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const {
+  topicData,
+  userData,
+  articleData,
+  commentData,
+} = require("../db/data/test-data");
+const request = require("supertest");
+
+beforeEach(() => {
+  return seed({ topicData, userData, articleData, commentData });
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET /api/topics", () => {
+  it("200: should return an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        body.topics.forEach((topic) => {
```

Comment:
As we're using a forEach here, could we check the length of the array before please? 👍